airbyte-integrations/connectors/source-float/manifest.yaml

```diff
@@ -0,0 +1,1155 @@
+          - number
+          - "null"
+      start_date:
+        type:
+          - string
+          - "null"
+      status_id:
+        type: number
+      status_type_id:
+        type:
+          - number
+          - "null"
+    required:
+      - status_id
+  time_off:
+    type: object
+    $schema: http://json-schema.org/schema#
+    additionalProperties: true
+    properties:
+      created:
+        type:
+          - string
+          - "null"
+      created_by:
+        type:
+          - number
+          - "null"
+      end_date:
+        type:
+          - string
+          - "null"
+      full_day:
+        type:
+          - number
+          - "null"
+      modified:
+        type:
+          - string
+          - "null"
+      modified_by:
+        type:
+          - number
+          - "null"
+      people_ids:
+        type:
+          - array
+          - "null"
+        items:
+          type:
+            - number
+            - "null"
+      repeat_state:
+        type:
+          - number
+          - "null"
+      start_date:
+        type:
+          - string
+          - "null"
+      status:
+        type:
+          - number
+          - "null"
+      timeoff_id:
+        type: number
+      timeoff_notes:
+        type:
+          - string
+          - "null"
+      timeoff_type_id:
+        type:
+          - number
+          - "null"
+      timeoff_type_name:
+        type:
+          - string
+          - "null"
+    required:
+      - timeoff_id
+  timeoff-types:
+    type: object
+    $schema: http://json-schema.org/schema#
+    additionalProperties: true
+    properties:
+      active:
+        type:
+          - number
+          - "null"
+      approval_required:
+        type:
+          - number
+          - "null"
+      color:
+        type:
+          - string
+          - "null"
+      timeoff_type_id:
+        type: number
+      timeoff_type_name:
+        type:
+          - string
+          - "null"
+    required:
+      - timeoff_type_id
+  clients:
+    type: object
+    $schema: http://json-schema.org/schema#
+    additionalProperties: true
+    properties:
+      client_id:
+        type: number
+      name:
+        type:
+          - string
+          - "null"
+    required:
+      - client_id
+  phases:
+    type: object
+    $schema: http://json-schema.org/draft-07/schema#
+    additionalProperties: true
+    properties: {}
+  project-tasks:
+    type: object
+    $schema: http://json-schema.org/draft-07/schema#
+    additionalProperties: true
+    properties: {}
+  milestones:
+    type: object
+    $schema: http://json-schema.org/draft-07/schema#
+    additionalProperties: true
+    properties: {}
+  allocations:
+    type: object
+    $schema: http://json-schema.org/draft-07/schema#
+    additionalProperties: true
+    properties: {}
+  logged-time:
+    type: object
+    $schema: http://json-schema.org/draft-07/schema#
+    additionalProperties: true
+    properties: {}
```

Comment:
@bala-ceg can you check why these streams aren't with the correct schema?

Reply:
I checked and fixed the streams with incorrect schema